packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -97,14 +102,38 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
     />
   );
 
-  const meTextStyle = mergeStyles({ color: theme.palette.neutralTertiary }, styles?.me);
-  const contextualMenuStyle = mergeStyles({ background: theme.palette.neutralLighterAlt }, styles?.menu);
-  const menuButtonContainerStyle = mergeStyles({
-    root: {
-      '&:hover': { background: 'none' },
-      '&:active': { background: 'none' }
-    }
-  });
+  const meTextStyle = useMemo(
+    () => mergeStyles({ color: theme.palette.neutralTertiary }, styles?.me),
+    [theme.palette.neutralTertiary, styles?.me]
+  );
+  const contextualMenuStyle = useMemo(
+    () => mergeStyles({ background: theme.palette.neutralLighterAlt }, styles?.menu),
+    [theme.palette.neutralLighterAlt, styles?.menu]
+  );
+  const iconsContainerWidth = useMemo(
+    () => (styles?.iconContainer as CSSProperties)?.width ?? '3.5rem',
```

Comment:
No need to memoize string (Only object references should be memoized to avoid rerenders)